### DISCUSSION TITLE
Plans for Sympa Release 6.2.74

### DISCUSSION THREAD

**Discussion by racke** [sympa-community/sympa#1673] [Release works]
Tentative date for next beta: 20th July

**Comment by ikedas** [comment]
I'd like to merge the following PRs with changes in message translation and corrections in documentation ahead of time:
#1584 , #1585 , #1599 , #1650 , #1655 , #1664 , #1653 , #1675 .

**Comment by ikedas** [comment]
I am in favor of merging all PRs currently labeled [ready] into the next release (or beta).

**Comment by racke** [comment]
OK, please go ahead.

**Comment by ikedas** [comment]
I have [suggested](https://github.com/sympa-community/sympa/discussions/1631#discussioncomment-6068755) to use Release drafter. However, I don't think I'll have time to try it for a while.  @racke , could you please create the `pr/relrease-6.2.74` branch for release note?

**Comment by racke** [comment]
Yes,  I created that branch without the typo :wink:

**Comment by ikedas** [comment]
I think all PRs currently tagged [ready] should be included in the next release. If none have objection, @racke, please put them in the 6.2.74 milestone.

**Comment by ikedas** [comment]
If you won't in some days, I'll put them in the milestone and merge it.

**Reply by racke** [reply]
I would do this next week and plan a release in December.

**Comment by ikedas** [comment]
OK.  I merged #1733 in advance since it updates translation catalog.  The all others marked [ready] are ready!

**Comment by ikedas** [comment]
I'll merge submitted PRs to solve issues by UIUC (#1754) to confirm if they work as expected.

**Comment by ikedas** [comment]
When will the next release be?

@racke, I’ll update `NEWS.md` once the PRs that have `ready` tags are merged.

(Note that I’ll merge #1827 by myself, because it will affect the translation catalog.)

**Comment by racke** [comment]
Moving into the flat of my girlfriend. Next slot would be 16th May.

**Comment by ikedas** [comment]
@racke, can you find the time?

**Comment by racke** [comment]
Planned for this week.

**Comment by ikedas** [comment]
If there are no problems, I will merge the PRs with [ready] tag.

@racke , might we like to see see the next release ... in this year at least?

**Comment by racke** [comment]
Yes, please merge the PRs with the ready tag.  Concerning the release .... I will try me best.

**Comment by ikedas** [comment]
I have merged a few PRs that seem to have a large impact, so I'll run some integrated tests on my end.

racke, where is the burden on the release process? For exmaple, would the autogenerating release notes eliminate it?

**Comment by morenonardelliunitn** [comment]
Hi @ikedas can i ask you if there is a scheduled new releases of Sympa in the next mounths?

**Comment by ikedas** [comment]
I’m planning to release a new version of Sympa in this or next week.

**Comment by ikedas** [comment]
Done!
